Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputActionsProfile.cs

```diff
@@ -11,6 +12,9 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Input Actions Profile", fileName = "MixedRealityInputActionsProfile", order = 1)]
     public class MixedRealityInputActionsProfile : ScriptableObject
     {
+        private readonly string[] inputActionsMRTKDefaults = { "Select", "Menu", "Grip", "Pointer", "Walk", "Look", "Interact", "Pickup", "Inventory", "ConversationSelect" };
```

Comment:
would be a bit easier to read if each element were on a separate line

Reply:
#Grumble, only a default :P